fill.go

```diff
@@ -123,6 +123,22 @@ func fillListeners(impl any, get func(name string) (net.Listener, string, error)
 	return nil
 }
 
+// isIdentifier returns whether the provided string is a valid Go identifier,
```

Comment:
Do we want this to be a Go identifier, or a valid listener name? If it is the latter, the comment is misleading.

Reply:
The latter. I renamed things and added some clarifying comments.